src/shifter.c

```diff
@@ -551,7 +551,7 @@ static void _usage(int status) {
     printf("\n"
         "Usage:\n"
         "shifter [-h|--help] [-v|--verbose] [--image=<imageType>:<imageTag>]\n"
-        "    [--entry] [-V|--volume=/path/to/bind:/mnt/in/image[:<flags>][,...]]\n"
+        "    [--entry] [-V|--volume=/path/to/bind:/mnt/in/image[:<flags>][;...]]\n"
```

Comment:
this is incorrect

see Alan's original suggestion:

[-V|--volume=/path/to/bind:/mnt/in/image[:[,...]][;...]]

which is correct.

the comma is for separating flags, the semicolon for mounts

Reply:
okay